policy/feature-flags.md

```diff
@@ -6,63 +6,137 @@ Product and release need to track bugs whose visibility is controlled through a
 
-It's decided that this should be controlled behind a preference, `browser.newtabpage.default.tabbycat`:
+If it's decided to disable the feature, then it should be turned off in the nightly build and the `behind-pref` flag set to `off`. The bug's comments should explain why that decision was released. Once the defects have been resolved, then `behind-pref` can be reset to `nightlyNN+2`.
 
-*   A comment is filed listing the name of the preference
-*   The `behind-pref` flag is set to +
-*   The `qe-verify` flag is set to ?, requesting QA's attention
+Once the feature has been verfied by QA then:
 
-![Screenshot of Bugzilla with tracking pane open for editing displaying the behind-pref and qe-verify flags](/public/images/feature-flags-editing-in-bmo.png)
+- The bug for the feature should be moved to RESOLVED VERIFIED status
+- The bug should be enabled in Beta once Release Management approves
+- the `behind-pref` flag is updated to `betaNN+1` 
 
-By default, the preference enabling the feature **should not** be set.
+The feature now *rides the trains* to release, and on merge day, `behind-pref` is set to `nightlyNN`. The bug is then considered completed. 
 
-A patch lands in mozilla-central with the code for the feature controlled by the preference, then:
+When the next ESR is released, the `behind-pref-esr` field should be set to the version where it was relased. 
 
-*   The bug's status is moved to RESOLVED and resolution to FIXED.
-*   QA agrees to take on verfication, setting `qe-verify` to `+`
-*   The feature is tested with and without the flag set and confirmed to work
-*   QA moveds the bug's resolution to VERIFIED
-*   Otherwise the patch is backed out to be landed again later, and the bug REOPENED
+### Questions 
 
+#### What if we turn off the feature in the main release?
 
-Once the bug for the feature has reached RESOLVED VERIFIED, then the lead for the feature which may be an engineer, a program manager, or a product manager must confirm with the Nightly Release Manager, and then enable it.
+The main bug's `behind-pref` value should be reset to the releases it's still on, `betaNN+1` or `nightlyNN+2`; or to `off`, and the bug's status set to REOPENED.
 
-*   The patch to flip the pref on should be attached to the "Make Tabby Cats the new default tab experience" bug
+The bug to turn off the feature must be a dependency of the main bug.
 
-If a decision is made to uplift the feature to another release, then RelMan and product must decide when to enable the feature. If they do decide to enable it then:
+#### What if we want to hold a feature over a release cycle and not promote it?
 
-*   The patch to turn the pref on which has been attached to the "Make Tabby Cats the new default tab experience" bug should be requested to be uplifted to the appropriate train
+On merge day, the `behind-pref` flag would change from the previous to the new Beta or Nightly value, and remain preffed off in other versions.
```

Comment:
As I mentioned above, I think the flag should be the target for release not the actual current release value to avoid this maintance burden.